code/game/gamemodes/changeling/powers/transform.dm

```diff
@@ -7,17 +7,138 @@
 	req_human = 1
 	max_genetic_damage = 3
 
+//changeling clothing items
+/obj/item/clothing/head/changeling

```

Comment:
are you sure that having it be one type won't cause issues when things try to read variables that arent there?